server/index.mjs

```diff
@@ -22,10 +22,12 @@ app.get("/weather", (req, res) => {
     units: "imperial",
   });
   const url = `https://api.openweathermap.org/data/2.5/weather?${params}`;
+  //const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${APIKEY}`
   console.log(url);
   fetch(url)
     .then((res) => res.json())
     .then((data) => {
+      console.log(data)
       res.send({ data });
     })
     .catch((err) => {
```

Comment:
Related to above, you should probably return this error to the client so you can show the user something went wrong.